src/MyQConnector/Service/BridgeService.cs

```diff
@@ -0,0 +1,103 @@
+using System;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using Microsoft.Extensions.Configuration;
+
+using Serilog;
+
+using Innovatrics.SmartFace.Integrations.AccessController.Notifications;
+using Innovatrics.SmartFace.Integrations.MyQConnectorNamespace.Models;
+using Innovatrics.SmartFace.Integrations.MyQConnectorNamespace.Factories;
+
+
+namespace Innovatrics.SmartFace.Integrations.MyQConnectorNamespace.Services
+{
+    public class BridgeService : IBridgeService
+    {
+        private readonly ILogger logger;
+        private readonly IConfiguration configuration;
+        private readonly IMyQConnectorFactory MyQConnectorFactory;
```

Comment:
myQConnectorFactory (C# ma variable ako camelCase - https://www.freecodecamp.org/news/snake-case-vs-camel-case-vs-pascal-case-vs-kebab-case-whats-the-difference/ )

Reply:
dakujem za info, dalsie miesta upravene fixnute